clients/benchmarks/client.cpp

```diff
@@ -48,14 +48,19 @@ using namespace std::literals;
 #include "testing_trsv.hpp"
 
 // Template to dispatch testing_gemm_ex for performance tests
-// When Ti == void or complex, the test is marked invalid
+// When Ti == void or complex or Ti == To == Tc == bfloat16, the test is marked invalid
 template <typename Ti, typename To = Ti, typename Tc = To, typename = void>
```

Comment:
I do not see a test for Ti == complex. Can the comment be updated if there is no test for complex.